joomla/joomla-cms
# [4.2] MediaVersion use Cache instead of Database (reduce 1 db query)

### Summary of Changes

Change getMediaVersion to use Infinity Cache instead of doing 1 useless DB query every time.

Reduce 1 db query for production installation, and 2 db query in debug mode.

### Testing Instructions

Apply patch and make sure MediaVersion added to every script, and all works as before.

Go to "clear cache" page and remove `_media_version` group, then check MediaVersion in to every script again, it should be new now.

### Documentation Changes Required

None
